scripts/qa_crowbarsetup.sh

```diff
@@ -2956,6 +2956,13 @@ function do_one_proposal
     # extract them for the proposal creation, but pass them to update_one_proposal
     local proposaltypemapped=$proposaltype
     proposaltype=${proposaltype%%+*}
+
+    # generic skip for proposals
+    if eval [[ \$want_${proposal}_proposal = 0 ]] ; then
```

Comment:
`eval` always scares me, and it's not necessary in this case, you can do something like

    skip_var="want_${proposal}_proposal"
    if [[ ${!skip_var} = 0 ]]; then